src/_Typography.sass

```diff
@@ -11,57 +11,57 @@ h3,
 h4,
 h5,
 h6
-	font-weight: 300
+	font-weight: $heading-font-weight
 	margin-bottom: 2.0rem
 	margin-top: 0
 
 h1
-	font-size: 4.0rem
-	letter-spacing: -.1rem
-	line-height: 1.2
+	font-size: $h1-font-size
+	letter-spacing: $heading-letter-spacing
+	line-height: $h1-line-height
 
 h2
-	font-size: 3.6rem
-	letter-spacing: -.1rem
-	line-height: 1.25
+	font-size: $h2-font-size
+	letter-spacing: $heading-letter-spacing
+	line-height: $h1-line-height
```

Comment:
@jackmcpickle Please correct line-height for headings h2-h6. They are all refering to $h1-line-height.